screener.config.js

```diff
@@ -28,15 +30,28 @@ const screenerConfig = {
   // screenshot every example in maximized mode
   states: glob
     .sync('docs/src/examples/**/*.tsx', { ignore: ['**/index.tsx', '**/*.knobs.tsx'] })
-    .map(examplePath => {
-      const { name: nameWithoutExtension, base: nameWithExtension } = path.parse(examplePath)
+    .reduce((states, examplePath) => {
+      const { name: nameWithoutExtension, base: nameWithExtension, dir } = path.parse(examplePath)
       const rtl = nameWithExtension.endsWith('.rtl.tsx')
+      const url = `http://localhost:8080/maximize/${_.kebabCase(nameWithoutExtension)}/${rtl}`
```

Comment:
```suggestion
      const url = `http://${config.server_host}:${config.server-host}/maximize/${_.kebabCase(nameWithoutExtension)}/${rtl}`
```

We have this vars in config, let's use them